src/orders/app.py

```diff
@@ -42,52 +41,37 @@ def create_order(event: Dict[str, Any], context: LambdaContext) -> Dict[str, Any
         is_auth = doorman.auth_user()
         if is_auth is False:
             raise AuthError("User is not allow to create a new order")
-        order_errors = []
 
         body = doorman.get_body_from_request()
         new_order_data = HIBerryOrder(**body)
 
         logger.info(
             f"Processing order for: {new_order_data.client_name} at {new_order_data.delivery_address}"
         )
-        lat_and_long = None
-        if new_order_data.geolocation is None:
-            logger.info("Input did not include geolocation data, invoking Geolocaction Service")
-            location_service = Geolocation()
-            lat_and_long = location_service.get_lat_and_long_from_street_address(
-                str_address=new_order_data.delivery_address
-            )
-        else:
-            logger.info("Reading geolocation data from input")
-            lat_and_long = new_order_data.geolocation.__dict__
-            # lat_and_long = None
-
-        if lat_and_long is None:
-            logger.info("Geolocation Data is missing, adding to the list of errors")
-            order_errors.append(
-                {
-                    "code": "ADDRESS_NEEDS_GEO",
-                    "value": "Order requires geolocation coordinates to be updated manually",
-                }
-            )
-            lat_and_long = {}
 
-        # ToDo validate Shopify order price vs owns price
-        builder = OrderHelper()
+        builder = OrderHelper(new_order_data.__dict__)
         order_db_data = builder.build_order(
-            order_data=new_order_data.__dict__,
             username=username,
-            geolocation=lat_and_long,
-            errors=order_errors,
         )
         dao = OrderDAO()
-        order_created = dao.create_order(order_db_data)
-        assigned_driver = order_created['payload']['driver']
-        logger.info(f"Order received and created with status {order_db_data['status']} and for driver {assigned_driver}")
-        output_data = {"status": order_db_data["status"], "assigned_driver": assigned_driver}
-        return doorman.build_response(
-            payload=output_data, status_code=201
-        )
+        create_response = dao.create_order(order_db_data)
+        
+        if create_response["status_code"] ==  201:
+            order_status = order_db_data["status"]
+            assigned_driver = order_db_data['driver']
+            
+            logger.info(f"Order received and created with status {order_status} and for driver {assigned_driver}")
+            output_data = {"status": order_db_data["status"], "assigned_driver": assigned_driver}
```

Comment:
```suggestion
            output_data = {"status": order_db_data["status"], "assigned_driver": assigned_driver, "errors": order_errors}
```
we need to return order_errors so the front end can map an order as error